native/cocos/audio/android/AudioEngine-inl.cpp

```diff
@@ -61,15 +68,23 @@ using namespace cc; //NOLINT
 namespace {
 AudioEngineImpl *gAudioImpl = nullptr;
 int outputSampleRate = 44100;
-int bufferSizeInFrames = 192;
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
+int              bufferSizeInFrames = 192;
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+// TODO(hack) : There is currently a bug in the opensles module,
+// so openharmony must configure a fixed size, otherwise the callback will be suspended
+int              bufferSizeInFrames = 2048;
+#endif
 
 void getAudioInfo() {
-    JNIEnv *env = JniHelper::getEnv();
-    jclass audioSystem = env->FindClass("android/media/AudioSystem");
-    jmethodID method = env->GetStaticMethodID(audioSystem, "getPrimaryOutputSamplingRate", "()I");
-    outputSampleRate = env->CallStaticIntMethod(audioSystem, method);
-    method = env->GetStaticMethodID(audioSystem, "getPrimaryOutputFrameCount", "()I");
-    bufferSizeInFrames = env->CallStaticIntMethod(audioSystem, method);
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
+    JNIEnv *  env         = JniHelper::getEnv();
+    jclass    audioSystem = env->FindClass("android/media/AudioSystem");
+    jmethodID method      = env->GetStaticMethodID(audioSystem, "getPrimaryOutputSamplingRate", "()I");
+    outputSampleRate      = env->CallStaticIntMethod(audioSystem, method);
+    method                = env->GetStaticMethodID(audioSystem, "getPrimaryOutputFrameCount", "()I");
+    bufferSizeInFrames    = env->CallStaticIntMethod(audioSystem, method);
+#endif
```

Comment:
if you add a macro here, `outputSampleRate` and `method` will become empty, and it might get into trouble.
Give it a correct value because we don't want to ruin the function's behavior.

Reply:
What value is correct? If we don't assign a value, use the default value. Isn't this default value the correct value?